java/buildconf/ivy/ivy-suse.xml

```diff
@@ -112,6 +108,7 @@
         <dependency org="suse" name="tomcat-servlet-4.0-api" rev="9.0.36" />
 
         <!-- Compilation and testing -->
+        <dependency org="org.apache.velocity" name="velocity-engine-core" rev="2.2" transitive="false"/>
```

Comment:
didn't you submit the packages to have it available from "suse" ?

Reply:
I have the submission at hand for Uyuni, but still a few things to investigate for HEAD. Just upgrading `velocity-engine` and `velocity-master` is not enough, we need at least a `guava` that is not built using kit and greater than 27. The one in `languages:java` is 30.x.

I also need to figure out what requires `velocity-tools` and `velocity-dvsl` since I haven't upgraded the first one and the second has been dropped upstream.

Reply:
Just fixed that one to point to my branch project to far, will need to remove that commit before merging once the SRs are approved